lib/api/v3/work_packages/work_package_representer.rb

```diff
@@ -593,6 +593,20 @@ def spent_time=(value)
           # noop
         end
 
+        def duration=(value)
+          unless OpenProject::FeatureDecisions.work_packages_duration_field_active?
+            raise NoMethodError,
+                  "#{WorkPackage.human_attribute_name(:duration)} #{I18n.t('activerecord.errors.messages.does_not_exist')}"
+          end
```

Comment:
I am surprised, that this is necessary. I would have thought that the `if` on the duration [would ignore the property when rendering but also when parsing](https://trailblazer.to/2.1/docs/representable.html#representable-function-api-if).

Reply:
You're right, it is not necessary as the contract definition takes care of it, removed.
0bb487ab269afa8c4b0420e88e4b46e072dd348a